cmd/analytics/analytics.go

```diff
@@ -8,7 +8,11 @@ import (
 	"github.com/spf13/viper"
 )
 
-func CmdRunEventProperties(cmd *cobra.Command, name string) map[string]interface{} {
+func CmdRunEventProperties(
+	cmd *cobra.Command,
+	name string,
+	overrides map[string]interface{},
```

Comment:
Add overrides so we can pass in an optional "action: help". We could make this functional options if we want to use it for more than the one-off case this introduces.